crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1313,7 +1368,7 @@ impl World {
     ///
     /// # Safety
     /// The value referenced by `value` must be valid for the given [`ComponentId`] of this world
-    /// `component_id` must exist in this [`World`]
+    /// `component_id` must exist in this [`World`].
```

Comment:
This safety invariant is not required anymore as far as i can tell

Reply:
same thing about `insert_non_send_by_id`

Reply:
No this is still accurate. We do not call `Components::init_resource` at all before this. `component_id` must be valid before this call.

Reply:
it will panic not be ub if the component id isnt valid